USAGEGUIDE.md

```diff
@@ -0,0 +1,263 @@
+## Table of Contents
```

Comment:
Is it worth having a title and short description for this page? 🤔 
(shouldn't be a blocker for this PR)